aspnetcore/security/app-secrets.md

```diff
@@ -31,53 +31,43 @@ For example, if you create a new ASP.NET Core web app with individual user accou
+The Secret Manager tool provides a general mechanism to store sensitive data for development work outside of your project tree. The Secret Manager tool is a project tool that can be used to store secrets for a [.NET Core](https://microsoft.com/net/core) project during development. With the Secret Manager tool you can associate app secrets with a specific project and share them across multiple projects.
 
 >[!WARNING]
 > The Secret Manager tool does not encrypt the stored secrets and should not be treated as a trusted store. It is for development purposes only. The keys and values are stored in a JSON configuration file in the user profile directory.
 
-### Installing the Secret Manager tool
+### Installing the Secret Manager tool 
 
-* Add `Microsoft.Extensions.SecretManager.Tools` to the `tools` section of the *project.json* file and run `dotnet restore`. 
+Add `Microsoft.Extensions.SecretManager.Tools` tools to the *.csproj* file and run 
+`dotnet restore`. 
 
-* Test the Secret Manager tool by running the following command:
+[!code-xml[Main](app-secrets/sample/UserSecrets/csproj.xml?highlight=17)]
 
-  <!-- literal_block {"ids": [], "xml:space": "preserve"} -->
+Test the Secret Manager tool by running the following command:
 
-  ```
-  dotnet user-secrets -h
+```
+dotnet user-secrets -h
      ```
 
-> [!NOTE]
-> When any of the tools are defined in the project.json file, you must be in the same directory in order to use the tooling commands.
-
 The Secret Manager tool will display usage, options and command help.
 
-The Secret Manager tool operates on project specific configuration settings that are stored in your user profile. To use user secrets the project must specify a `userSecretsId` value in its *project.json* file. The value of `userSecretsId` is arbitrary, but is generally unique to the project.
-
-* Add a `userSecretsId` for your project in its *project.json* file:
-
-<!-- literal_block {"ids": [], "linenos": false, "xml:space": "preserve", "language": "json", "highlight_args": {"hl_lines": [2]}} -->
+> [!NOTE]
+> You must be in the same directory of the *.csproj* file to run tools defined in the *.csproj* file.
 
-```json
-{
-"userSecretsId": "aspnet-WebApp1-c23d27a4-eb88-4b18-9b77-2a93f3b15119",
+The Secret Manager tool operates on project specific configuration settings that are stored in your user profile. To use user secrets the project must specify a `UserSecretsId` value in its *.csproj* file. The value of `UserSecretsId` is arbitrary, but is generally unique to the project.
 
-"dependencies": {
-```
+Add a `UserSecretsId` for your project in the  *.csproj* file :
 
-* Use the Secret Manager tool to set a secret. For example, in a command window from the project directory enter the following:
+[!code-xml[Main](app-secrets/sample/UserSecrets/csproj.xml?range=7-9&highlight=2)]
```

Comment:
You aaren't highlighting the PropertyGroup tags  ... when the whole block needs to be inserted it might be better not to highlight anything rather than highlight one line.